tests/test_other.py

```diff
@@ -988,6 +988,30 @@ def test_link_group_bitcode(self):
     self.run_process([EMCC, 'out.o'])
     self.assertContained('Hello', self.run_js('a.out.js'))
 
+  def test_link_single_ar_bitcode(self):
+    create_test_file('1.c', r'''
+int f(void);
+int main() {
+  f();
+  return 0;
+}
+''')
+    create_test_file('2.c', r'''
+#include <stdio.h>
+int f() {
+  printf("World\n");
+  return 0;
+}
+''')
+
+    self.run_process([EMCC, '-o', '1.o', '-c', '1.c'])
+    self.run_process([EMCC, '-o', '2.o', '-c', '2.c'])
+    self.run_process([EMAR, 'crs', 'lib.a', '1.o', '2.o'])
+    self.run_process([EMCC, '-r', '-o', 'lib.o', '-Wl,--whole-archive,--start-group', 'lib.a', '-Wl,--end-group,--no-whole-archive'])
```

Comment:
You don't need the start-group/end-group here.

Reply:
Thanks. I've removed the `--no-whole-archive` part as well.